base/utils/R/sensitivity.R

```diff
@@ -296,6 +296,15 @@ write.sa.configs <- function(defaults, quantile.samples, settings, model,
               file = file.path(settings$rundir, run.id, "README.txt"), 
               sep = "")
           
+          # I check to make sure the path under the met is a list. if it's specified what met needs to be used in 'met.id' under sensitivity analysis of pecan xml we used that otherwise, I use the first met.
+          if (is.list(settings$run$inputs$met$path)){
+            if (is.null(settings$sensitivity.analysis$met.id))  settings$sensitivity.analysis$met.id <- 1
+            settings$run$inputs$met$path <- settings$run$inputs$met$path [settings$sensitivity.analysis$met.id]
```

Comment:
Small thing, but would be good if you made the `else` explicit here and if you didn't have a space in `path [`

Reply:
@mdietze I corrected the `path [`.
In the logic of the code I didn't think there is a need for an `else` statement (if it's `else` then it's in my desired format/shape).  
Let me know if you think this definitely needs to have an `else`.